packages/snap/package.json

```diff
@@ -29,7 +29,7 @@
     "test": "jest",
     "typia:generate": "typia generate --input src/typia-templates --output src/typia-generated --project tsconfig.json",
     "generate-ether-types": "typechain --target=ethers-v5 'abis/**/*.json' --out-dir src/types/contracts",
-    "prepublishOnly": "yarn build"
+    "prepublishOnly": "yarn && yarn build"
```

Comment:
Also add a `preversion` hook that builds the snap and does `git add snap.manifest.json` to make sure that the shasum in the commit will match the shasum in the npm package